buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -6,6 +6,9 @@ object Dependencies {
     const val kotlinVersion = "1.6.10"
     const val springCoreVersion = "5.3.14"
     const val springBootVersion = "2.6.2"
+    const val springBoot25Version = "2.5.8"
+    const val springBoot24Version = "2.4.13"
+    const val springBoot23Version = "2.3.12.RELEASE"
```

Comment:
-- korean
여러 프로젝트에서 사용되는 것이 아니기 때문에 사용되는 곳에서만 버전을 명시하고 Dependencies 에는 제외해도 좋을 것 같습니다.

-- english
Since it is not used in other modules, it is okay to specify the version only where it is used and exclude it from Dependencies.

Reply:
-- korean
반영 하였습니다.

--english
applied